src/main/java/org/jboss/xavier/sanitizer/XavierPayloadSanitizer.java

```diff
@@ -0,0 +1,169 @@
+package org.jboss.xavier.sanitizer;
+
+import com.jayway.jsonpath.*;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class XavierPayloadSanitizer
+{
+    private static String inputFileName;
+    private static String outputFileName;
+    private static String issuesConditionsJSONFileName;
+
+    public static void main(final String[] args)
+    {
+        List<String> argsList = Arrays.asList(args);
+        argsList.forEach(arg -> {
+            if( arg.equals("--input"))
+            {
+                inputFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+            if( arg.equals("--output"))
+            {
+                outputFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+
+            if( arg.equals("--issues"))
+            {
+                issuesConditionsJSONFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+        });
+
+        String jsonFileToBeSanitized = readInputFile(inputFileName, "Input file to sanitize");
+
+        String jsonIssuesConditions = readInputFile(issuesConditionsJSONFileName, "Issues Conditions file ");
+        List<String> vmsIssues = splitConditions(jsonIssuesConditions,"###vms");
+        List<String> hostsIssues = splitConditions(jsonIssuesConditions,"###hosts");
+        List<String> clustersIssues = splitConditions(jsonIssuesConditions,"###clusters");
+
+        String outputJsonString = sanitize(jsonFileToBeSanitized, vmsIssues, hostsIssues,clustersIssues);
+
+        writeStringToFile(outputFileName, outputJsonString);
+
+    }
+
+    private static String readInputFile(String inputFile, String description)
+    {
+        if (inputFile == null)
+        {
+            throw new RuntimeException(description + "path not specified");
+        }
+        try
+        {
+            return new String(Files.readAllBytes(Paths.get(inputFile)), StandardCharsets.UTF_8);
+        }
```

Comment:
I think we should add some code to verify that `String inputFile` exists. Something like:

```
Path inputPath = Paths.get(inputFile);
if (!Files.exists(inputPath))
{
    throw new RuntimeException(description + "path does not exists. Path=" + inputPath.toString());
}

try
{
    return new String(Files.readAllBytes(inputPath), StandardCharsets.UTF_8);
}
```